BuildSupport.sh

```diff
@@ -2,17 +2,8 @@
 
 set -e
 
-# "nuget restore" fails if local package source directories don't exist.
-mkdir -p NuPkgs/Support
-
-# Final output directory of NuPkgs.
-NUPKG_DIR=`pwd`/NuPkgs/Support
-# Build configuration to build/pack.
-BUILD_CONFIGURATION=Release
-
 # Forces sourcelink to work during the build.
 export CI=true
 
-dotnet restore Src/Support/GoogleApisClient.sln
```

Comment:
In samples I've had to add explicit force restore to all projects. Else, ocasionally there were failures of dependencies not found. After I added the explicit restore, all such failures dissapeared. See here https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1469#discussion_r699987353.
I have no good explanation for it.

Reply:
I believe that was an issue with earlier .NET SDKs. I haven't seen it in a while. I suggest we go with this as-is now, but we can add the restore later if it becomes a problem again.